tests/Php73/Php72Test.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Tests\Php73;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Polyfill\Php73\Php73 as p;
+
+/**
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ *
+ * @covers Symfony\Polyfill\Php73\Php73::<!public>
+ */
+class Php73Test extends TestCase
+{
+    /**
+     * @covers Symfony\Polyfill\Php73\Php73::is_countable
+     */
+    public function testIsCountable()
+    {
+        $this->assertTrue(is_countable([1, 2, 3]));
+        $this->assertTrue(is_countable(new \ArrayIterator(['foo', 'bar', 'baz'])));
+        $this->assertTrue(is_countable(new \ArrayIterator()));
+        $this->assertFalse(is_countable(new \stdClass()));
```

Comment:
please add a case with generator

Reply:
@keradus Is this possible with 5.x tests?

Reply:
even if not, it shall still be tested